### ISSUE TITLE
invalid code generated for `extern const` 

### ISSUE THREAD

**Issue by arnetheduck** [nim-lang/c2nim#241]
Header:
```c
extern const int xxx_symbol;
```
Command:
```bash
$ c2nim --version
0.9.18

$ c2nim --header --importc --nep1 test.h
```
Output:
```nim
let xxxSymbol* {.header: "test.h".}: cint
```

several problems with the generated definition:

* it's missing `importc: "xxx_symbol"`
* it should be a `var` (because `let` needs an initializer which in this case is not provided in nim (it's done in the C library typically)

Of course, the main issue here is that Nim has no feature that expresses the concept of an `extern const` in general, so `var` is the closest that we can get.

**Comment by Araq** [comment]
`let x {.importc.} <init missing>` is Nim's notion of "extern const". IIRC this was added for this reason.

**Comment by arnetheduck** [comment]
> is Nim's notion of "extern const"

right, I was testing this with nim 1.2 where it's not allowed - it's also still missing the actual `importc`
